### DISCUSSION TITLE
OpenDTU on Battery: etwa 3 Stunden nach Sonnenuntergang werden alle Daten gelöscht

### DISCUSSION THREAD

**Discussion by Fokker-1** [tbnobody/OpenDTU#2423] [Q&A]
Guten Morgen.

ich betreibe 2x HMS-1000-2t an einer openDTU on battery.
Die FW ist aktuell, das System läuft störungsfrei. DPL und alle anderen Battery-Funktionen sind inaktiv geschaltet.

Seit etwa 2 Wochen tritt allerdings folgendes Verhalten auf:

Etwa 3 Stunden nach Sonnenuntergang (Anzeigefelder sind rot, keine Produktion mehr), setzen sich die bis dahin angezeigten Werte auf Null zurück bei:

"Inverter Gesamtertrag insgesamt" und "Inverter Gesamtertrag heute".

"Leistung Netz" funktioniert auch weiterhin, "Inverter Gesamtleistung" ist natürlich auf Null.

Kann mir hier bitte jemand helfen?

Danke im Voraus und viele Grüße
Werner

**Comment by ms49434** [comment]
Support für openDTU-OnBattery gibt es eigentlich hier: https://github.com/hoylabs/OpenDTU-OnBattery

Du kannst aber trotzdem mal die Einstellungen deines Systems überprüfen, insbesondere die eingestellte Zeitzone, die aktuelle Systemzeit, die NTP Einstellungen und die erweiterten Einstellungenen der Wechselrichter.

**Comment by Fokker-1** [comment]
Hallo,

vielen Dank.

Ja, das hatte ich alles geprüft, passt alles.
Unter Deinem Link hatte ich jetzt mal gelesen, aber nichts Passendes gefunden - bin aber auch nicht so fit in der SW.

Ich könnte ja vielleicht auch mal versuchen, einen älteren JSON-Key einzulesen. Da habe ich einen, der ist aber auf dem Stand der vorherigen FW.
Geht das in einer solchen Kombination oder muss der JSON immer von der identischen FW stammen?

Danke und Gruß
Werner

**Comment by Fokker-1** [comment]
Das Einlesen dann hier unetr "Wiederherstellen":

![Confug-Verw_oBatt](https://github.com/user-attachments/assets/5709dc93-5567-450a-b97e-9e0b418de217)

**Comment by Juergen2453** [comment]
Du bist hier immer noch Falsch. Gehe in den Kanal für support-opendtu-onbattery.
https://discord.com/channels/984173303147155506/1230866028414767168

**Comment by Fokker-1** [comment]
Hallo,

ich hatte jetzt testweise mal eine openDTU Fusion aktiviert, da läuft alles problemlos.

Deren Einstellungen hatte ich dann in die onBatt übernommen:
mal funktionierte es, mal aber auch nicht. Alle sehr eigentümlich.

Na ja, ich werde mal weiter suchen.
Das dauert alles sehr lange, da man ja immer einen Nachtzyklus warten muss, um ein Ergebnis zu erhalten.

Noch mal danke an alle und Grüße
Werner
